server/src/test/java/org/opensearch/index/compositeindex/datacube/startree/StarTreeTestUtils.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.compositeindex.datacube.startree;
+
+import org.opensearch.index.compositeindex.datacube.startree.aggregators.numerictype.StarTreeNumericType;
+
+public class StarTreeTestUtils {
+
+    public static Double toStarTreeNumericTypeValue(Long value, StarTreeNumericType starTreeNumericType) {
+        try {
+            return starTreeNumericType.getDoubleValue(value);
+        } catch (Exception e) {
+            throw new IllegalStateException("Cannot convert " + value + " to sortable aggregation type", e);
```

Comment:
Why are we converting this (since this is a test util, don't we need the actual exception to be propagate into our tests)?